src/js/constants.js

```diff
@@ -22,6 +22,32 @@ export const MediaUIEvents = {
   UNREGISTER_MEDIA_STATE_RECEIVER: 'unregistermediastatereceiver',
 };
 
+export const MediaStateChangeEvents = {
+  MEDIA_AIRPLAY_UNAVAILABLE: 'mediaairplayunavailablechange',
+  MEDIA_PIP_UNAVAILABLE: 'mediapipunavailablechange',
+  MEDIA_PAUSED: 'mediapausedchange',
```

Comment:
I like the consistency here, and also it's the crux of the decision. It would be odd to have both this and `play`/`pause`/`playing` events, to mimic the video element API. This is making me consider not mimicking the video element. That requires a little more thought and discussion.